langchain-core/src/messages/index.ts

```diff
@@ -16,4 +16,5 @@ export {
   type InvalidToolCall,
   isToolMessage,
   isToolMessageChunk,
+  type ToolCall,
```

Comment:
This conflicts with the `ToolCall` declared in `base.ts` - you should import it like this:

```ts
import { type ToolCall } from "@langchain/core/messages/tool";
```